### ISSUE TITLE
Bazel crash on bad --experimental_remote_platform_override

### ISSUE THREAD

**Issue by ulfjack** [bazelbuild/bazel#3153]
```
java.lang.IllegalArgumentException: Failed to parse --experimental_remote_platform_override
	at com.google.devtools.build.lib.remote.RemoteSpawnStrategy.<init>(RemoteSpawnStrategy.java:97)
	at com.google.devtools.build.lib.remote.RemoteActionContextProvider.<init>(RemoteActionContextProvider.java:44)
	at com.google.devtools.build.lib.remote.RemoteModule.executorInit(RemoteModule.java:99)
	at com.google.devtools.build.lib.buildtool.ExecutionTool.<init>(ExecutionTool.java:193)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:168)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:374)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:124)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:104)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:541)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:322)
	at com.google.devtools.build.lib.runtime.CommandExecutor.exec(CommandExecutor.java:53)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:863)
	at com.google.devtools.build.lib.server.GrpcServerImpl.access$2100(GrpcServerImpl.java:106)
	at com.google.devtools.build.lib.server.GrpcServerImpl$4$1.run(GrpcServerImpl.java:929)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.google.protobuf.TextFormat$ParseException: 1:1: Input contains unknown fields and/or extensions:
1:1:	google.devtools.remoteexecution.v1test.Platform.entry
	at com.google.protobuf.TextFormat$Parser.checkUnknownFields(TextFormat.java:1403)
	at com.google.protobuf.TextFormat$Parser.merge(TextFormat.java:1426)
	at com.google.protobuf.TextFormat$Parser.merge(TextFormat.java:1342)
	at com.google.devtools.build.lib.remote.RemoteSpawnStrategy.<init>(RemoteSpawnStrategy.java:95)
	... 16 more
```

**Comment by katre** [comment]
@ulfjack, if you are looking into this please assign it to yourself, otherwise I'll try and figure out who to hand it off to.

**Comment by ola-rozenfeld** [comment]
We just switched to V1 API, the format of the flag is different now (as are many other flags). Please use 'properties' instead of 'entry'. The full flag changes are:

remote_worker -> remote_executor
grpc_timeout_seconds -> remote_timeout
remote_allow_local_fallback -> remote_local_fallback
tls_cert -> tls_certificate
tls_authority_override -> experimental_tls_authority_override
auth_credentials_json -> auth_credentials

Removed flags:

grpc_max_batch_size_bytes
grpc_max_chunk_size_bytes
grpc_max_batch_inputs

**Comment by ulfjack** [comment]
@ola-rozenfeld I figured it out eventually. But we should still improve the error message in this case.

**Comment by buchgr** [comment]
Fixed by #7168
